src/caliper/AllocTracker.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
```

Comment:
If this header is part of the public API - and it seems to be as it's included in DataTracker.h - it needs to be in include/caliper/.